app/models/app_meta_data.rb

```diff
@@ -3,8 +3,8 @@ class AppMetaData < MetaData
 
   def self.seed
     transaction do
-      env = ENV['SOURCES_ENV']
-      raise "No SOURCES_ENV set, cannot continue seeding." unless env
+      # Default to CI
+      env = ENV['SOURCES_ENV'] || "ci"
```

Comment:
there will be mandatory ENV_NAME parameter in ClowdApp CRD so I'm wondering if we can join these env values.
But it would need to have some default because the ENV_NAME can be "dev-env" , "ephemeral"(?) etc.
So I want to discuss about it

Reply:
maybe we can just make it default to `ci` if it isn't found. I like that `ENV_NAME` is going to be present - we can just remove `SOURCES_ENV` from the e2e-deploy deploymentconfig that way.